.gitignore

```diff
@@ -1,10 +1,5 @@
 __pycache__
-.DS_Store
```

Comment:
you can keep it in the `.gitignore` (it is a file autogenerated by OSX), it just shouldn't be in the repo

Reply:
solved in https://github.com/e-mission/op-admin-dashboard/pull/7/commits/62307f87190e87e9ee7e390b41681120d79f613f